bdpy/opendata/openneuro.py

```diff
@@ -52,16 +131,24 @@ def makedata(src, source_type='bids_daily', output_dir='./output', root_dir='./'
         trg_anat_raw = os.path.join(trg_anat_dir, '%s_ses-anatomy_T1w_raw.nii.gz' % subject)
         trg_anat_defaced = os.path.join(trg_anat_dir, '%s_ses-anatomy_T1w.nii.gz' % subject)
         __create_dir(trg_anat_dir)
-        if not dry_run and not os.path.exists(trg_anat_defaced):
-            ext = os.path.splitext(src_anat)[1]
-            if ext == '.gz':
-                shutil.copy2(src_anat, trg_anat_raw)  # FIXME: convert to nii.gz
-            else:
-                convert_command = 'mri_convert %s %s' % (src_anat, trg_anat_raw)
-                os.system(convert_command)
-            deface_command = 'pydeface %s --outfile %s' % (trg_anat_raw, trg_anat_defaced)
-            os.system(deface_command)
-            os.remove(trg_anat_raw)
+
+        if anatomy:
+            if not dry_run and not os.path.exists(trg_anat_defaced):
+                ext = os.path.splitext(src_anat)[1]
+                if ext == '.gz':
+                    shutil.copy2(src_anat, trg_anat_raw)  # FIXME: convert to nii.gz
+                else:
+                    convert_command = 'mri_convert %s %s' % (src_anat, trg_anat_raw)
+                    result = subprocess.run(convert_command.split(" "), capture_output=True, text=True)
+                    if result.returncode != 0:
+                        raise RuntimeError(f"{result.stdout}")
+                    
+                # Due to environment-dependent issues, it is possible to specify whether to run deface.
+                if run_deface:
```

Comment:
`run_deface=False` is generally a danger option. I don't think we should change the function interface of a public repository solely because of temporary inconsistencies in the internal environment.